mailpoet/assets/js/src/automation/templates/components/from-scratch.tsx

```diff
@@ -0,0 +1,97 @@
+import { Dispatch, SetStateAction, useCallback, useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { Hooks } from 'wp-js-hooks';
+import { Icon, plusCircleFilled } from '@wordpress/icons';
+import { PremiumModal } from '../../../common/premium_modal';
+import { Notice } from '../../../notices/notice';
+
+type FromScratchPremiumModalProps = {
+  showModal: boolean;
+  setShowModal: Dispatch<SetStateAction<boolean>>;
+};
+
+function FromScratchPremiumModal({
```

Comment:
The automation listing as well as the templates page did enqueue wp-component styles. I removed this in [d7140cf](https://github.com/mailpoet/mailpoet/pull/4362/commits/d7140cff91cb6251986e53e0a63400c677b78232)

Reply:
Ahh! And the removal is safe? Because we do use `@wordpress/components` so we need to load their styles somehow. In the editor, [we bundle them now](https://github.com/mailpoet/mailpoet/pull/4225/files) so I guess we need to bundle them also anywhere we use them and also within the modal?

It feels like removing these lines could break some existing styles, if `@wordpress/components` are used...

Reply:
> In the editor, we bundle them now

We do so also in mailpoet-plugin.scss, which is loaded throughout the screens: https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/assets/css/src/mailpoet-plugin.scss#L24

I removed them now in 9fe003aa3 from the mailpoet-automation-editor.scss as they are not needed there.

What we witness is an update of the styles ([PR](https://github.com/WordPress/gutenberg/pull/40781)), This update affects Gutenberg version>13.7 while in [WordPress core the current Gutenberg version is 13.0](https://developer.wordpress.org/block-editor/contributors/versions-in-wordpress/).

So, with our current way, we divert from core and "enqueue" the latest styles (or the styles of the components version, we require). Not entirely sure, if that's the best way (thinking of 3rd parties potentially enqueuing wp-components)

Reply:
Ahh, thanks for the explanation! Yeah, I guess at this point it's correct to "divert" in the styles because we're diverting also with the JS (bundling them instead of using [@wordpress/dependency-extraction-webpack-plugin](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-dependency-extraction-webpack-plugin/)) which we may fix in the future. It would require supporting only few recent versions of WP, and testing the UI across them. For now, we picked the "easier" way and did what's done in the form editor.